cpp/README.md

```diff
@@ -52,3 +55,34 @@ equivalent of the `input` field of an `evaluate` message. This can be useful for
 running C++ implementations or a debugger or a profiler. You must usually obtain
 `input.json` from the log file produced by `gradbench`, as most evals are not
 able to produce such files directly.
+
+## Multithreading
+
+The primal functions have been multithreaded with OpenMP when applicable. We try
+to write the code such that it will run well when compiled for both sequential
+and multithreaded execution, ideally without implementing each function twice.
+
+OpenMP is fairly well suited for this purpose, as pragmas are (almost) ignored
+when `-fopenmp` is not passed to the C compiler, although we have encountered
+some wrinkles.
+
+### Reductions
+
+One problem is that Enzyme does not support the OpenMP reduction clause. This
```

Comment:
oh I didn't know this wasn't supported, open an issue? It was supported for some version of clang/libopenmp

Reply:
There is/was already an issue: https://github.com/EnzymeAD/Enzyme/issues/1246

Reply:
Yeah so that was closed so we added support, so it not working is news to us

Reply:
I think that issue was closed because it was migrated to a discussion: https://github.com/EnzymeAD/Enzyme/discussions/1251

If I write a program with a straightforward +-reduction I get the same error as in that issue. I can of course still open a new issue; let me just put together a self-contained program.